src/ott/solvers/nn/losses.py

```diff
@@ -0,0 +1,95 @@
+# Copyright OTT-JAX
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from types import MappingProxyType
+from typing import Any, Literal, Mapping, Optional, Union
+
+import jax
+import jax.numpy as jnp
+
+from ott.geometry import costs, pointcloud
+from ott.solvers.linear import sinkhorn
+
+
+def monge_gap(
+    samples: jnp.ndarray,
+    mapped_samples: jnp.ndarray,
+    cost_fn: Optional[costs.CostFn] = None,
+    epsilon: Optional[float] = None,
+    relative_epsilon: Optional[bool] = None,
+    scale_cost: Union[bool, int, float, Literal["mean", "max_cost",
+                                                "median"]] = 1.0,
+    sinkhorn_kwargs: Mapping[str, Any] = MappingProxyType({}),
```

Comment:
I think you can remove the `sinkhorn_kwargs` and just use `kwargs` for `sinkhorn.solve()`, since you expose most (if not all) useful `PointCloud` arguments explicitly.